src/Psalm/Config.php

```diff
@@ -1560,8 +1560,10 @@ public function visitStubFiles(Codebase $codebase, Progress $progress = null)
 
         $phpstorm_meta_path = $this->base_dir . DIRECTORY_SEPARATOR . '.phpstorm.meta.php';
 
-        if (file_exists($phpstorm_meta_path)) {
+        if (is_file($phpstorm_meta_path)) {
             $stub_files[] = $phpstorm_meta_path;
+        } elseif (is_directory($phpstorm_meta_path)) {
+            $progress->debug('phpstorm meta path not included in stub files list as it is a directory' . "\n");
```

Comment:
The docs say files in this folder have the same format, so they could all be added to `$stub_files` perhaps?

Reply:
🤷‍♂️ *does not use phpstorm*

Reply:
How deep does the folder go?

Reply:
I'm not a PHPStorm user either. I suppose *all* files should be read, regardless of the folder depth.

Reply:
@weirdan https://confluence.jetbrains.com/display/PhpStorm/PhpStorm+Advanced+Metadata

> 2016.2 + -> you can have multiple separate files in FOLDER named `.phpstorm.meta.php`